plugins/scss/index.js

```diff
@@ -154,14 +159,54 @@ const writer = function(thisPackage, outputDir) {
       file = path.resolve(outputDir, 'combined.scss');
       args.push(file);
       console.log('Writing ' + file);
-      promises.push(concat(scssEntries, file));
+      promises.push(concat.concatAsync(scssEntries, file));
     }
 
     file = path.resolve(outputDir, 'node-sass-args');
     console.log('Writing ' + file);
     promises.push(fs.writeFileAsync(file, args.join(' ')));
 
-    return Promise.all(promises);
+    return new Promise(function(resolve, reject) {
+      resolve(Promise.all(promises).then(function() {
```

Comment:
I know we don't have a good lint rule for this, but please move `.then()` and other chained promise calls to their own line to match up with project conventions.

Reply:
sure